.github/workflows/ci.yml

```
@@ -146,3 +146,17 @@ jobs:
         env:
           # This secret is automatically injected by GitHub
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: 🚨 Run Lighthouse
+        run: yarn lighthouse
+        env:
+          secrets: '{"JWT_SECRET":"JWT_SECRET","OKTA_CLIENT_SECRET":"OKTA_CLIENT_SECRET"}'
+          LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
+          MPDX_USERNAME: ${{ secrets.MPDX_USERNAME }}
+          MPDX_PASSWORD: ${{ secrets.MPDX_PASSWORD }}
+          PREVIEW_URL: 'https://pr-${{ github.event.number }}.d3dytjb8adxkk5.amplifyapp.com'
```

As I'm sure you know, this will need updated to the new preview urls once your PR merges. The following might work, but it will fail if the PR doesn't have the "Preview Environment" label. Maybe we skip the lighthouse step if there isn't a preview environment?

```suggestion
          PREVIEW_URL: 'https://${{ env.GITHUB_HEAD_REF }}.d3dytjb8adxkk5.amplifyapp.com'
```

---

Daniel suggested we make this run on comment since it takes so long, but we can certainly ensure it can't run if the preview label isn't applied as well.

---

I'm good with that or however you want to handle it.